rust/resolver/src/error.rs

```diff
@@ -13,7 +13,7 @@ pub enum Error {
     HttpClient(#[from] reqwest::Error),
     /// Error making rpc request via solana client
     #[error("solana rpc-client error: {0}")]
-    Rpc(#[from] client_error::Error),
+    Rpc(#[from] Box<client_error::Error>),
```

Comment:
Q: why this change was needed?

Reply:
@GabrielePicco because this variant is too big https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err